v5/cli/conductor.rst

```diff
@@ -1,3 +1,79 @@
-===============
-Conductor (CLI)
-===============
+====================
+Conductor User Guide
+====================
+
+Introduction
+============
+
+Conductor is PROS's templating and project management facility. It is
+responsible for resolving versions of the PROS kernel and user libraries. It also
+provides information about a PROS project to other PROS facilities, like
+providing the target platform (e.g. V5 or Cortex) to the upload utility.
+
+There are a few concepts central to Conductor's functionality:
+
+- Templates: Templates are a description of a set of files. They have a name,
+  version, supported target, and possibly other metadata. Templates typically
+  come in one of 3 forms: remote, local, and installed. Remote templates are
+  templates that can be downloaded. Local templates are a downloaded template

```

Comment:
plurals need to agree:

> Local templates are downloaded templates...

or

> A local template is a downloaded template...

Reply:
I'd go with the latter because it seems that's what you go with for the other two types (or change all three)